draft/draft-scharf-tcpm-yang-tcp.xml

```diff
@@ -101,14 +101,35 @@
       means to configure the parameters listed in this document, which are
       outside the scope of this document.</t>
 
-      <t>This specification is orthogonal to <xref target="RFC4022">Management
+      <t>This specification does not deprecate the <xref target="RFC4022">Management
```

Comment:
I am not sure if a YANG model can deprecate a SNMP MIB. In that sense, don't you think the previous wording made more sense?